app/pages/search/SearchPageContainer.js

```diff
@@ -30,13 +30,17 @@ export class UnconnectedSearchPageContainer extends Component {
   }
 
   render() {
-    const { isFetching, resources, searchFilters } = this.props;
-
+    const { isFetching, resultsCount, resources, searchFilters } = this.props;
+    const searchResultsText = resultsCount === 1 ? 'täyttävä tila' : 'täyttävää tilaa';
```

Comment:
- [ ] The searchResultsText could contain the whole text and not just the end of it.